patches/@changesets+apply-release-plan+6.0.4.patch

```diff
@@ -0,0 +1,25 @@
+diff --git a/node_modules/@changesets/apply-release-plan/dist/apply-release-plan.cjs.dev.js b/node_modules/@changesets/apply-release-plan/dist/apply-release-plan.cjs.dev.js
```

Comment:
See https://github.com/stylelint/stylelint/pull/6299#discussion_r952719715

Reply:
I did a quick search for issues or pull requests to request such a feature in the [changesets/changesets](https://github.com/changesets/changesets) repo, but I could not... 😓 Do you have any information?

Honestly, I'm worried that we may need to test or modify this patch with each update of the `@changeset/*` packages. At least, I wish there would be a feature request issue and a pull request using someone's fork version, instead of applying and managing the patch in our repository.

Reply:
When we agree with these changes and merge, I'll pr back, I want to guarded it works as expected for us first.

Reply:
@ybiquitous I just raised https://github.com/changesets/changesets/issues/926